src/main/webapp/app/exercises/modeling/participate/modeling-submission.component.ts

```diff
@@ -401,6 +400,24 @@ export class ModelingSubmissionComponent implements OnInit, OnDestroy, Component
         }
     }
 
+    /**
+     * Find "Unreferenced Feedback" item for Result, if it exists.
+     */
+    get unreferencedFeedback(): Feedback[] | undefined {
+        if (this.assessmentResult && this.assessmentResult.feedbacks && Array.isArray(this.assessmentResult.feedbacks)) {
+            return this.assessmentResult.feedbacks.filter((feedbackElement) => feedbackElement.reference == undefined && feedbackElement.type === FeedbackType.MANUAL_UNREFERENCED);
+        }
+    }
+
+    /**
+     * Find "Referenced Feedback" item for Result, if it exists.
+     */
+    get referencedFeedback(): Feedback[] | undefined {
+        if (this.assessmentResult && this.assessmentResult.feedbacks && Array.isArray(this.assessmentResult.feedbacks)) {
+            return this.assessmentResult.feedbacks.filter((feedbackElement) => feedbackElement.reference !== undefined);
```

Comment:
```suggestion
            return this.assessmentResult.feedbacks.filter((feedbackElement) => feedbackElement.reference != undefined);
```

Reply:
Does it make a difference here? I wouldn't say type-checking is unnecessary. Maybe someone uses the reference "undefined" :P

Reply:
It does make a difference, using just `!=` also accounts for the variable being `null`.

Reply:
Alright, i am giving in :D